doc/source/index.rst

```diff
@@ -11,107 +11,59 @@ PyFluent documentation |version|
-
-- Launch the Fluent solver in serial or parallel and connect to already-running
-  Fluent sessions. For more information, see :ref:`ref_user_guide_launch`.
-- Script using Fluent's meshing capabilities. For more information, see
-  :ref:`ref_meshing`.
-- Script using all of Fluent's TUI  (text user interface) commands. For more
-  information, see :ref:`ref_user_guide_tui_commands`.
-- Run more than one Fluent session asynchronously. For more information, see
-  :ref:`ref_execution_utils`.
-- Retrieve Fluent field data as numpy arrays for custom postprocessing using
-  standard Python packages such as `matplotlib <https://matplotlib.org/>`_. For
-  more information, see :ref:`ref_field_data`.
-- Register function callbacks on Fluent solver events such as when a case file
-  or data file is read or when the Fluent solver completes an iteration. For
-  more information, see :ref:`ref_events`.
-- Retrieve solver monitors such as residuals. For more information, see
-  :ref:`ref_monitors`.
-
-Documentation and issues
-------------------------
-Documentation for the latest stable release of PyFluent is hosted at
-`PyFluent documentation <https://fluent.docs.pyansys.com/version/stable/>`_.
-
-In the upper right corner of the documentation's title bar, there is an option for switching from
-viewing the documentation for the latest stable release to viewing the documentation for the
-development version or previously released versions.
-
-You can also `view <https://cheatsheets.docs.pyansys.com/pyfluent_cheat_sheet.png>`_ or
-`download <https://cheatsheets.docs.pyansys.com/pyfluent_cheat_sheet.pdf>`_ the
-PyFluent cheat sheet. This one-page reference provides syntax rules and commands
-for using PyFluent.
-
-On the `PyFluent Issues <https://github.com/ansys/pyfluent/issues>`_ page, you can create
-issues to report bugs and request new features. On the `PyFluent Discussions
-<https://github.com/ansys/pyfluent/discussions>`_ page or the `Discussions <https://discuss.ansys.com/>`_
-page on the Ansys Developer portal, you can post questions, share ideas, and get community feedback.
-
-To reach the project support team, email `pyansys.core@ansys.com <pyansys.core@ansys.com>`_.
-
-License and acknowledgments
----------------------------
-PyFluent is licensed under the MIT license.
-
-PyFluent makes no commercial claim over Ansys whatsoever. This library
-extends the functionality of Ansys Fluent by adding a Python interface
-to Fluent without changing the core behavior or license of the original
-software. The use of the interactive Fluent control of PyFluent requires a
-legally licensed local copy of Fluent.
-
-For more information on Fluent, see the `Ansys Fluent <https://www.ansys.com/products/fluids/ansys-fluent>`_
-page on the Ansys website.
-
-Project index
--------------
-
-* :ref:`genindex`
+
+- `PyFluent GitHub repository <https://github.com/ansys/pyfluent>`_
+- `Fluent documentation <https://www.ansys.com/products/fluids/ansys-fluent>`_
```

Comment:
I don't know how to go from that page to the actual Fluent documentation. Would linking directly to the help documentation here be better?
```suggestion
- `Fluent documentation <https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/prod_page.html?pn=Fluent>`_
```

Reply:
Changed to public doc link which doesn't require login